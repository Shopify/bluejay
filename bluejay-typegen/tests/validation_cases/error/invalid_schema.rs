#[bluejay_typegen::typegen([
    type Query {
        foo: Bar
    }
])]
mod schema {}

fn main() {}
