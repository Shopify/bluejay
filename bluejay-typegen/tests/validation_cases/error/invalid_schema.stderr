error: Error: Referenced type `Bar` does not exist
          ╭─[ <unknown>:1:19 ]
          │
        1 │ type Query { foo: Bar }
          │                   ─┬─
          │                    ╰─── No definition for referenced type
       ───╯
 --> tests/validation_cases/error/invalid_schema.rs:1:28
  |
1 |   #[bluejay_typegen::typegen([
  |  ____________________________^
2 | |     type Query {
3 | |         foo: Bar
4 | |     }
5 | | ])]
  | |_^
